Lib/test/test_unittest/testmock/testthreadingmock.py

```diff
@@ -133,9 +137,7 @@ def test_wait_failed_with_timeout_override(self):
 
         with patch(f"{__name__}.Something", waitable_mock):
             something = Something()
-            self.run_async(something.method_1, delay=0.1)
```

Comment:
@mariocj89 - this change and the ones below confuse me, are they intentional or just a result of a bad attempt of mine to resolve the conflicts?

Reply:
I've just rebased all change son top of main.